src/CMakeLists.txt

```diff
@@ -213,9 +213,20 @@ if(NOT "${PLATFORM}" STREQUAL "windows")
   execute_process(
     COMMAND "${PYTHON_EXECUTABLE}" "-c"
         "import capnp; import os; import sys; sys.stdout.write(os.path.dirname(os.path.dirname(capnp.__file__)))"
+        RESULT_VARIABLE src_pycapnp_include_result_code
         OUTPUT_VARIABLE src_pycapnp_include_dir)
 
+  if(src_pycapnp_include_result_code)
```

Comment:
This fixes a silent failure that I discovered as the result of DEVOPS-146 that used to manifest itself in a surprising build error much later in the build process.